src/net_processing.cpp

```diff
@@ -367,12 +367,10 @@ struct CNodeState {
     bool fPreferredDownload{false};
     //! Whether this peer wants invs or headers (when possible) for block announcements.
     bool fPreferHeaders{false};
-    //! Whether this peer wants invs or cmpctblocks (when possible) for block announcements.
-    bool fPreferHeaderAndIDs{false};
-    /**
-      * Whether this peer will send us cmpctblocks if we request them.
-      */
-    bool fProvidesHeaderAndIDs{false};
+    /** Whether this peer wants invs or cmpctblocks (when possible) for block announcements. */
+    bool m_requested_hb_cmpctblocks{false};
+    /** Whether this peer will send us cmpctblocks if we request them. */
+    bool m_provides_cmpctblocks{false};
```

Comment:
744011062864d2d6dd03a5c8c381807d41063ac5: Any reason this isn't done as a scripted diff? The only other change seems to be the comment change, which I believe will be touched again anyway when this is moved to Peer.

Reply:
I tried to split this PR into small commits after this comment: https://github.com/bitcoin/bitcoin/pull/20799#issuecomment-752197637. I'm not sure if there's a way to structure these changes that is going to satisfy everyone, so I'd prefer to leave this as is.